### ISSUE TITLE
Fails with psycopg2-binary

### ISSUE THREAD

**Issue by ipmb** [linkedin/shiv#8]
Running `shiv -o myproject .`

Here is the traceback:

```
Traceback (most recent call last):
  File "/Users/pete/.virtualenvs/saltdash-apCx4VH9/bin/shiv", line 11, in <module>
    sys.exit(main())
  File "/Users/pete/.virtualenvs/saltdash-apCx4VH9/lib/python3.6/site-packages/click/core.py", line 722, in __call__
    return self.main(*args, **kwargs)
  File "/Users/pete/.virtualenvs/saltdash-apCx4VH9/lib/python3.6/site-packages/click/core.py", line 697, in main
    rv = self.invoke(ctx)
  File "/Users/pete/.virtualenvs/saltdash-apCx4VH9/lib/python3.6/site-packages/click/core.py", line 895, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/Users/pete/.virtualenvs/saltdash-apCx4VH9/lib/python3.6/site-packages/click/core.py", line 535, in invoke
    return callback(*args, **kwargs)
  File "/Users/pete/.virtualenvs/saltdash-apCx4VH9/lib/python3.6/site-packages/shiv/cli.py", line 180, in main
    shared_object_map=map_shared_objects(site_packages),
  File "/Users/pete/.virtualenvs/saltdash-apCx4VH9/lib/python3.6/site-packages/shiv/cli.py", line 85, in map_shared_objects
    module_name, import_tag, extension = fullpath.parts[-1].split(".")
ValueError: too many values to unpack (expected 3)
```

The file causing the error is:

```
>>> import pdb; pdb.pm()
> /Users/pete/.virtualenvs/saltdash-apCx4VH9/lib/python3.6/site-packages/shiv/cli.py(85)map_shared_objects()
-> module_name, import_tag, extension = fullpath.parts[-1].split(".")
(Pdb) fullpath
PosixPath('/var/folders/nw/7ftbjmn50y9fy047fgyv4c3r0000gn/T/tmpdjxe_i80/site-packages/psycopg2/.dylibs/libcrypto.1.0.0.dylib')
```

I have a dependency on `psycopg2-binary==2.7.4`

**Comment by sixninetynine** [comment]
Hi!

Thanks for the report, @dan-blanchard actually pointed this bug out in #7 
Terribly sorry, especially because the map_shared_objects stuff is actually not even used anywhere! (it's some leftover code from a feature that was half-baked!)

https://github.com/linkedin/shiv/pull/9

**Comment by sixninetynine** [comment]
published `0.0.13` to pypi with the fix
